data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "entity_type": "work",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "description": "A commentary on the six days of Creation. The first of its kind, this style of commentary, called Hexaemeron, flourished in late antiquity, and later commentaries drew on Basil's as a model.",
+  "authority_files": [{
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
+    },
+    {
+      "id": "https://viaf.org/viaf/181934772",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. | Hexaemeron"
+    }
+  ],
+  "associated_dates": [{
+    "type": "creation",
+    "date_human_readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375"
+  }],
+  "language": "Greek",
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "associated_entities": [{
```

Comment:
We could also include ‘standardized’ incipit, prologue, rubric, etc. based on standard editions (this would be kind of like how we might define a primary role for persons). Would still need a source? Other incipits, etc. would be backlinked in display from the artifacts, and it may be unnecessary for our purposes to indicate a standard/normalized version. This is again an issue of scope and purpose of these authority files, which are for disambiguation purposes within the SMDL rather than providing exhaustive and normative reference info for the conceptual entities.